### PR TITLE
Add all compat data for HTMLInputElement

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Checks off HTMLInputElement from the list on #2467!

### CODE REVIEW

File: api/HTMLInputElement.json [hunk 1]
```diff
@@ -0,0 +1,2036 @@
...
+      },
+      "autocomplete": {
+        "__compat": {
+          "description": "<code>autocomplete</code> property",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: should be `true`.

File: api/HTMLInputElement.json [hunk 2]
```diff
@@ -0,0 +1,2036 @@
...
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: `true`

File: api/HTMLInputElement.json [hunk 3]
```diff
@@ -0,0 +1,2036 @@
...
+      },
+      "autofocus": {
+        "__compat": {
+          "description": "<code>autofocus</code> property",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: `true`

File: api/HTMLInputElement.json [hunk 4]
```diff
@@ -0,0 +1,2036 @@
...
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: `true`

File: api/HTMLInputElement.json [hunk 5]
```diff
@@ -1301,7 +1301,7 @@
             },
             "opera_android": {
               "version_added": null
-            },
+          true
             "safari": {
               "version_added": true
```
Comment (jpmedley) [inline api/HTMLInputElement.json]: Something went wrong here. There are other instances farther down. Please check throughout. 

Do you know about this: http://jsoneditoronline.org/#/

Comment (connorshea) [reply api/HTMLInputElement.json]: it's odd that the linter didn't catch this 🤔

Comment (vinyldarkscratch) [reply api/HTMLInputElement.json]: Oh haha, how’d this happen?  I think this was a part of my mistakes when manually editing the JSON myself, oops!

**Pull request merged**